client/state/login/utils.js

```diff
@@ -150,7 +166,7 @@ export const isRegularAccount = authAccountType => authAccountType === 'regular'
 export const isPasswordlessAccount = authAccountType => authAccountType === 'passwordless';
 
 export async function postLoginRequest( action, bodyObj ) {
-	const response = await fetch(
+	const response = await window.fetch(
```

Comment:
This change is added to silence a lint error.

Reply:
@sgomes would this be better as `globalThis.fetch` or is it good as-is?

(https://github.com/Automattic/wp-calypso/pull/38313)

Reply:
If it'll only ever run on the client, it's better to leave it as `window.fetch`. If it's meant to also run on the server in some cases, it'll need to be `globalThis.fetch`, or it'll fail.

Reply:
As far as I know, it's only meant to be run on the client.